specs/IsPointInDraggableRegion.md

```diff
@@ -0,0 +1,303 @@
+        }
+```
+
+# API Details
+<!-- TEMPLATE
+    The exact API, in IDL format for our COM API and
+    in MIDL3 format (https://learn.microsoft.com/uwp/midl-3/)
+    when possible.
+
+    Include every new or modified type but use // ... to remove any methods,
+    properties, or events that are unchanged.
+
+    For the MIDL3 parts, after running build-apiwriter, open the generated
+    `Microsoft.Web.WebView2.Core.idl` and find the new or modified portions
+    generated from your modifications to the COM IDL.
+
+    (GitHub's markdown syntax formatter does not (yet) know about MIDL3, so
+    use ```c# instead even when writing MIDL3.)
+
+    Example:
+    
+```
+[uuid(B625A89E-368F-43F5-BCBA-39AA6234CCF8), object, pointer_default(unique)]
+interface ICoreWebView2Settings4 : ICoreWebView2Settings3 {
+  /// The IsPinchZoomEnabled property enables or disables the ability of 
+  /// the end user to use a pinching motion on touch input enabled devices
+  /// to scale the web content in the WebView2. It defaults to TRUE.
+  /// When set to FALSE, the end user cannot pinch zoom.
+  /// This API only affects the Page Scale zoom and has no effect on the
+  /// existing browser zoom properties (IsZoomControlEnabled and ZoomFactor)
+  /// or other end user mechanisms for zooming.
+  ///
+  /// \snippet SettingsComponent.cpp TogglePinchZooomEnabled
+  [propget] HRESULT IsPinchZoomEnabled([out, retval] BOOL* enabled);
+  /// Set the IsPinchZoomEnabled property
+  [propput] HRESULT IsPinchZoomEnabled([in] BOOL enabled);
+}
+```
+
+```c# (but really MIDL3)
+namespace Microsoft.Web.WebView2.Core
+{
+    runtimeclass CoreWebView2Settings
+    {
+        // ...
+
+        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Settings5")]
+        {
+            Boolean IsPinchZoomEnabled { get; set; };
+        }
+    }
+}
+```
+
+If you are introducing a WebView2 JavaScript API include the TypeScript
+definition of that API and reference documentation for it as well.
+You can use https://www.typescriptlang.org/play to verify your TypeScript
+
+```ts
+interface WebView extends EventTarget {
+    postMessage(message: any) : void;
+    hostObjects: HostObjectsAsyncRoot;
+    // ...
+}
+
+interface HostObjectsAsyncRoot {
+    cleanupSome() : void;
+    options: HostObjectsOptions;
+}
+
+interface HostObjectsOptions {
+    forceLocalProperties: string[];
+    log: (...data: any[]) => void;
+    shouldSerializeDates: boolean;
+    defaultSyncProxy: boolean;
+    forceAsyncMethodMatches: RegExp[];
+    ignoreMemberNotFoundError: boolean;
+}
+```
+
+-->
+## Win32 C++
+```cpp
+[uuid(42BF7BA5-917A-4C27-ADA1-EA6969854C16), object, pointer_default(unique)]
+interface ICoreWebView2StagingController : NULL {
+    [propget] HRESULT IsPointInDraggableRegion([out, retval] BOOL* value);
+}
+```
+## .Net/ WinRT
+```c#
+namespace Microsoft.Web.WebView2.Core {
+    runtimeclass CoreWebView2StagingController {
+        public bool IsPointInDraggableRegion(ref Point p);
```

Comment:
After running build-apiwriter, open the generated `Microsoft.Web.WebView2.Core.idl` and find the new or modified portions generated from your modifications to the COM IDL and copy them here. There should be an interface_name block in which your new method is defined. Your MIDL3 should have the interface_name block shown as well. But you can do // ... for the other unmodified parts of CoreWebView2Controller. I don't think it will have 'public' or 'ref'